src/main/java/net/dv8tion/jda/api/entities/Message.java

```diff
@@ -1504,9 +1533,6 @@ default boolean isFromGuild()
      *
      * @throws java.lang.UnsupportedOperationException
      *         If this is not a Received Message from {@link net.dv8tion.jda.api.entities.MessageType#DEFAULT MessageType.DEFAULT}
-     * @throws net.dv8tion.jda.api.exceptions.InsufficientPermissionException
-     *         If the MessageChannel this message was sent in was a {@link net.dv8tion.jda.api.entities.TextChannel TextChannel} and the
-     *         logged in account does not have {@link net.dv8tion.jda.api.Permission#MESSAGE_HISTORY Permission.MESSAGE_HISTORY} in the channel.
```

Comment:
If this endpoint requires this permission the check should be added, instead of removing this from the docs.

Reply:
Should the check be added to this PR, or separately? Also, should the change include `MessageChannel#retrieveReactionUsersById` methods?

Reply:
If the check isn't already there, it should be added here.